README.md

```diff
@@ -42,6 +42,7 @@ A set of SVG maps that can be installed via [npm](https://www.npmjs.com) or [yar
 * 🇺🇸 [USA](packages/usa)
 * 🇺🇸 [USA, Counties](packages/usa.counties)
 * 🇺🇸 [USA, Utah](packages/usa.utah)
+* 🇺🇿 [Uzbekistan, Regions](packages/uzbekistan)
```

Comment:
Are there smaller or larger administrative divisions (e.g. counties, departments...)?
If yes, could you rename the package `[Uzbekistan, Regions](packages/uzbekistan.regions)` please?
If no, could you change the displayed name `[Uzbekistan](packages/uzbekistan)` please?

Reply:
Basically, there are no smaller or larger administrative divisions. There is one autonomous republic in Uzbekistan but is it always come together with regions. Therefore i did not separate them.
I have i have removed ", Regions" part of the code and kept the package name as it is.